assets/src/application/ui/layout/espressoTable/EspressoDragSortingTable/styles.scss

```diff
@@ -0,0 +1,14 @@
+.table-drag-sorting {
```

Comment:
Styles from here(in the bottom): https://github.com/ant-design/ant-design/blob/master/components/table/demo/drag-sorting.md